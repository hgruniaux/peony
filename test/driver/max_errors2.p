//# RUN: peony -fmax-errors=0 %s

// -fmax-errors=0 allows an unlimited number of errors
% //~ ERROR: unexpected token, found '%'
% //~ ERROR: unexpected token, found '%'
% //~ ERROR: unexpected token, found '%'
