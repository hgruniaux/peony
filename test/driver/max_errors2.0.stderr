<source>:4:1: error: unexpected token, found '%'
<source>:5:1: error: unexpected token, found '%'
<source>:6:1: error: unexpected token, found '%'
